tests/test_apiclient.py

```diff
@@ -1220,6 +1220,17 @@ def test_find_framework_suppliers_with_statuses(self, data_client, rmock):
         assert result == {'supplierFrameworks': [{"agreementReturned": False}, {"agreementReturned": True}]}
         assert rmock.called
 
+    def test_find_framework_suppliers_no_declarations(self, data_client, rmock):
+        rmock.get(
+            'http://baseurl/frameworks/g-cloud-7/suppliers?with_declarations=false&',
```

Comment:
Why do we get the trailing `&` here? Tests for e.g. filtering by status don't get a trailing & on the url.